.travis.yml

```diff
@@ -22,7 +23,11 @@ addons:
 install:
   #- sudo pip install pymdown-extensions
 
+# uncomment the following lines to override the default test script
+#script:
+#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#  - julia --check-bounds=yes -e 'using Pkg; Pkg.add(pwd()); Pkg.build("MLDatasets"); Pkg.test("MLDatasets"; coverage=true)'
+
 after_success:
-  - julia -e 'cd(Pkg.dir("MLDatasets")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
-  - julia -e 'Pkg.add("Documenter")'
-  - julia -e 'cd(Pkg.dir("MLDatasets")); include(joinpath("docs", "make.jl"))'
+  - julia -e 'using Pkg, MLDatasets; cd(joinpath(dirname(pathof(MLDatasets)), "..")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
+  - julie -e 'using Pkg; Pkg.add("Documenter"); cd(joinpath(dirname(pathof(MLDatasets)), "..")); include(joinpath("docs", "make.jl"))'
```

Comment:
the coverage doesn't seem to work. Maybe we should try emulating https://github.com/fredrikekre/Literate.jl/blob/master/.travis.yml

Reply:
From Travis logs it looks like the coverage script was running, but then there were some network problems, so it failed to connect to Coveralls server.